.prettierignore

```diff
@@ -0,0 +1,5 @@
+node_modules
+dist
+docs
+.storybook
```

Comment:
why?

Reply:
same for docs, they're manually written, why not use prettier on them?

Reply:
You are right `.storybook` is written manually, thought it was autogenereted. But `docs` is autogenerated. It is the output of the storybook build.